bluesky/tools/trafficarrays.py

```diff
@@ -66,8 +66,10 @@ def create(self, n=1):
         for v in self.LstVars:  # Lists (mostly used for strings)
 
             # Get type
-            if len(v) > 0:
-                vartype = str(type(v[0])).strip("<type '").strip("'>")
+            vartype = None
+            lst = self.__dict__.get(v)
+            if len(lst) > 0:
+                vartype = str(type(lst[0])).split("'")[1]

```

Comment:
This is a bug. Need to get the list value corresponding to key `v`. And then get the type of the first element of the list.
`v[0]` is a single character!